src/style.css

```diff
@@ -0,0 +1,153 @@
+/* #region reset */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+* {
+  box-sizing: border-box;
+  --main-accent: #060b35;
+  --blue-accent: #00acdc;
+
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
+/* #endregion */
+
+/* #region container settings */
+.card-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start ;
+  align-items: center;
+
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  font-family: Roboto, sans-serif;
+}
+/* #endregion */
+
+/* #region content */
+.card-photo {
+  padding: 0;
+  height: 134px;
+  width: 160px;
+  margin-bottom: 40px;
+}
+
+.product-name {
+  margin-bottom: 4px;
+
+  font-family: inherit;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+
+  color: var(--main-accent);
+}
+
+.product-code {
+  line-height: 14px;
+  font-size: 11px;
+  font-weight: 400;
+  font-family: inherit;
+
+  color: #616070;
+}
+/* #endregion */
+
+/* #region rating */
+.product-rating {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 15px;
+}
+
+.product-rating__stars {
+  display: flex;
+  height: 16px;
+}
+
+.star {
+  background: url(images/star.svg) no-repeat center;
+  width: 16px;
+}
+
+.star-active {
+  background: url(images/star-active.svg) no-repeat center;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.product-rating__feedback {
+  margin: 0;
+  font-size: 11px;
+  font-weight: 400;
+  line-height: 14px;
+
+  color: var(--main-accent);
+}
+/* #endregion */
+
+/* #region price */
+.price-value {
+  margin: 23px 0 16px;
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  align-items: baseline;
+}
+
+.price-value__price {
+  font-size: 11px;
+  font-weight: 400;
+  line-height: 18px;
+
+  color: #616070;
+}
+
+.price-value__value {
+  font-size: 16px;
+  font-weight: bold;
+  line-height: 19px;
+  color: var(--main-accent);
+}
+/* #endregion */
+
+/* #region button */
+.buy-button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  height: 40px;
+
+  font-size: 14px;
+  font-weight: bold;
+  line-height: 16px;
+  border-radius: 5px;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: #fff;
+  background: #00acdc;
+}
+
+.buy-button:hover {
+  background-color: #fff;
+  border: 1px solid var(--blue-accent);
```

Comment:
add border to .buy-button, and change only border-color on hover, not the whole border

Reply:
You didn't fix that